src/main/java/frc/team2412/robot/subsystems/VisionSubsystem.java

```diff
@@ -73,66 +79,18 @@ public VisionSubsystem(SwerveDrivePoseEstimator initialPoseEstimator) {
 	}
 
 	public void updateEvent(NetworkTableEvent event) {
-		latestResult = photonCamera.getLatestResult();
-		updatePoseCache();
-		if (robotPose != null) {
-			lastTimestampSeconds = latestResult.getTimestampSeconds();
-			poseEstimator.addVisionMeasurement(robotPose.toPose2d(), lastTimestampSeconds);
+		latestPose = photonPoseEstimator.update();
+		if (latestPose.isPresent()) {
+			System.out.println(latestPose.get().estimatedPose);
```

Comment:
Ideally, log entries will have a signifier / some text to make sure the code location can be easily found.

Reply:
That wasn't supposed to be left in there, forgot to remove it